reflex/utils/prerequisites.py

```diff
@@ -276,6 +267,47 @@ def remove_existing_bun_installation():
         path_ops.rm(os.path.expandvars(constants.BUN_ROOT_PATH))
 
 
+def validate_and_install_node():
+    """Validate nodejs have install or not."""
+    if not check_node_version():
+        install_node()
```

Comment:
Just like we do for bun, we should also prompt users whether or not to install node